src/index.html

```diff
@@ -11,8 +11,69 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The stylesheet link is pointing to an SCSS file (`index.scss`). Ensure that this file is being processed into CSS, or change the link to point to the resulting CSS file (e.g., `index.css`).